lms/lmsdb/models.py

```diff
@@ -143,6 +153,44 @@ def random_password(cls, stronger: bool = False) -> str:
     def get_notifications(self) -> Iterable['Notification']:
         return Notification.fetch(self)
 
+    def notes_to_show(self) -> List[Dict[str, Any]]:
```

Comment:
Prefer returning a DB object. If you want it as a list of dicts, make it in models/notes.py

Reply:
The name of the function isn't clear. The user can't distinguish between `notes`  and `notes_to_show`

Reply:
I don't understand your first comment - it's similar to as_dicts method of Exercise.